tgui/packages/tgui/interfaces/OreSilo.tsx

```diff
@@ -0,0 +1,199 @@
+import { classes } from 'common/react';
+import { capitalize } from 'common/string';
+import { useState } from 'react';
+
+import { useBackend } from '../backend';
+import {
+  Box,
+  Icon,
+  Image,
+  LabeledList,
+  NoticeBox,
+  Section,
+  Stack,
+  Tabs,
+  Tooltip,
+} from '../components';
+import { Window } from '../layouts';
+import { MaterialAccessBar } from './Fabrication/MaterialAccessBar';
+import { Material } from './Fabrication/Types';
+
+type Machine = {
+  ref: string;
+  name: string;
+  icon: string;
+  onHold: boolean;
+  location: string;
+};
+
+type Log = {
+  rawMaterials: string;
+  machineName: string;
+  areaName: string;
+  action: string;
+  amount: number;
+  time: string;
+  noun: string;
+};
+
+type OreSiloData = {
+  SHEET_MATERIAL_AMOUNT: number;
+  materials: Material[];
+  machines: Machine[];
+  logs: Log[];
+};
+
+export const OreSilo = (props: any, context: any) => {
+  const { act, data } = useBackend<OreSiloData>();
+  const { SHEET_MATERIAL_AMOUNT, machines, logs } = data;
+
+  const [currentTab, setCurrentTab] = useState(0);
+
+  return (
+    <Window title="Ore Silo" width={400} height={600}>
```

Comment:
Increase the width of this window so the material bar isn wrapped around the window, it looks ugly and unaligned, just make the same size as your techfab window

Reply:
i thought it looked fine

Reply:
Nothing wrong with it true,  but people are used to seeing the material bar displayed in a single line in all machines.
No need to break that continuity now